### PR TITLE
update systemd/units/init.sls

### PR DESCRIPTION
**Pull request by fhe3-dh**
Fixed iteration over pillars
Fixed path to jinja template

### CODE REVIEW

File: systemd/units/init.sls [hunk 1]
```diff
@@ -1,10 +1,10 @@
-{% for unittype, units in pillar['systemd:units'].iteritems()  %}
+{% for unittype, units in pillar['systemd'].iteritems()  %}
```
Comment (epcim) [inline systemd/units/init.sls]: I miss example in example Pilař or readme. 

Quite sure this First line si wrong. I Ha logical sena to have

Systemd:
     Unit:{}
     Network:{}

Comment (fhe3-dh) [reply systemd/units/init.sls]: pillar.example is there (without units) and it works as expected. Not sure what is missing.

Comment (fhe3-dh) [reply systemd/units/init.sls]: I tested it on debian-9 with vagrant and also in production. No problem on both

**Comment by aboe76** [comment]
@epcim you are right thanks for fixing this

**Pull request merged**